pyrealm_build_data/phenology/python_implementation.py

```diff
@@ -406,14 +406,14 @@ def run_length_encode(values: NDArray) -> tuple[NDArray[np.int_], NDArray]:
 # Extract the variables needed to run the model
 fortnightly_outputs = xr.Dataset(
     data_vars={
-        "tc_mean": fortnight_means["tc"],
-        "vpd_mean": fortnight_means["vpd"],
-        "patm_mean": fortnight_means["patm"],
-        "co2_mean": fortnight_means["co2"],
-        "ppfd_mean": fortnight_means["ppfd"],
-        "precip_molar_sum": fortnight_sum["precip_molar"],
+        "tc_mean": fortnight_means["TA_F"],
+        "vpd_mean": fortnight_means["VPD_F"],
+        "patm_mean": fortnight_means["PA_F"],
+        "co2_mean": fortnight_means["CO2_F_MDS"],
+        "ppfd_mean": fortnight_means["PPFD"],
+        "precip_molar_sum": fortnight_sum["P_F_MOLAR"],
         "year": fortnight_means.time.dt.year,
-        "PMod_gpp_smstress": fortnightly_gpp_smstress,
+        # "PMod_gpp_smstress": fortnightly_gpp_smstress,
```

Comment:
Delete this if it's no longer needed?

Reply:
I'll come back to this in the next PR, which will need to update this implementation.